README.md

```diff
@@ -107,6 +110,14 @@ After selecting the forms, press ‘Send’ to initiate the connection process s
 
 Tapping on ‘Send’ button opens up the screen which will initiate the communication process and start the wifi hotspot so that receiver can connect to it and start receiving the forms.
 
+In the Skunkworks-crow, we have two ways to exchange forms, Wifi hotspot and the bluetooth. You can choose one as the default transferring method in the settings page. 
```

Comment:
Forms can be exchanged using either Wifi Hotspot or Bluetooth. The default preference for the transferring method can be modified through the settings page.